src/components/ec-table-pagination/ec-table-pagination.vue

```diff
@@ -149,6 +149,7 @@ const isDesktop = useMediaQuery('(min-width: 768px)');
 
 const canShowPageSize = computed(() => (props.isPageSizeHidden === false || (props.isPageSizeHidden === undefined && isDesktop.value)));
 const canShowCustomInfo = computed(() => (props.isCustomInfoHidden === false || (props.isCustomInfoHidden === undefined && isDesktop.value)));
+const canShowTextEllipsis = computed(() => (!isDesktop.value && (!canShowCustomInfo.value || !canShowPageSize.value)));
```

Comment:
what if this text will be always ellipsis?

Reply:
Tables where we don't have the responsive changes applied yet will look like this:
![image](https://github.com/Ebury/chameleon/assets/77344353/21ce7514-0a3d-4f07-a086-c0e135c3f4f8)

So I think it would be better to not use ellipsis in these cases